languages/rust/generic/Parse_rust_tree_sitter.ml

```diff
@@ -2063,23 +2057,17 @@ and map_meta_arguments (env : env) ((v1, v2, v3, v4) : CST.meta_arguments) :
   in
   let _comma = Option.map (fun tok -> token env tok) v3 in
   let _rparen = token env v4 (* ")" *) in
-  args
+  (_lparen, args, _rparen)
 
-and map_meta_item (env : env) ((v1, v2) : CST.meta_item) =
+and map_meta_item (env : env) tok ((v1, v2) : CST.meta_item) : G.attribute =
   let name = map_path_name env v1 in
-  let path = H2.dotted_ident_of_name name in
-  let value =
-    Option.map
-      (fun x ->
-        match x with
-        | `EQ_lit (v1, v2) ->
-            let _equals = token env v1 (* "=" *) in
-            let lit = map_literal env v2 in
-            MetaItemLiteral lit
-        | `Meta_args x -> MetaItemMetaArgs (map_meta_arguments env x))
-      v2
-  in
-  (path, value)
+  match v2 with
+  | None -> NamedAttr (tok, name, (G.sc, [], G.sc))
```

Comment:
instead of wrapping in `(G.sc, _, G.sc)`, you can use `fb`, defined above! (which stands for "fake bracket")